### PR TITLE
@webpack-blocks/babel: Support for Babel 7.x

### PR DESCRIPTION
**Pull request by dmitmel**
[Babel 7.x](https://babeljs.io/blog/2017/09/12/planning-for-7.0) will be [released soon](https://babeljs.io/blog/2017/12/27/nearing-the-7.0-release), so it will be great if the Babel block fully supports the next Babel version.

### CODE REVIEW

File: packages/babel/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # @webpack-blocks/babel - Changelog
 
+## Next Release
+
+- Add support for `babel-loader` 8.x
```
Comment (vlad-zhukov) [inline packages/babel/CHANGELOG.md]: Should it say "Support babel 7."?

File: packages/babel/package.json [hunk 2]
```diff
@@ -19,12 +19,9 @@
...
   "bugs": "https://github.com/andywer/webpack-blocks/issues",
   "dependencies": {
-    "babel-loader": "^7.1.1"
+    "babel-loader": "^8.0.0-beta"
   },
   "devDependencies": {
     "@webpack-blocks/core": "^1.0.0-rc"
-  },
-  "peerDependencies": {
-    "babel-core": ">=6"
```
Comment (vlad-zhukov) [inline packages/babel/package.json]: We still require `@babel/core` for this to work.

**Comment by vlad-zhukov** [comment]
This is a breaking change because it won't work for babel 6, and I'm not a fan of supporting both v6 and v7 in this block as it will require some complex logic.

This PR can be only merged once we cut v1 of `webpack-blocks` and start working on v2, and `babel@7` releases a stable version.

File: packages/babel/CHANGELOG.md (hunk 1 above)
Comment (vlad-zhukov) [reply packages/babel/CHANGELOG.md]: Or even explicitly say that it only works for babel 7.

**Comment by zcei** [comment]
> it will require some complex logic
You have an example for this?

I built a - rather trivial - Babel 7 setup over in the planned new version of axios, and it "just worked"™️: https://github.com/axios/axios/commit/7779279529963736eab79d7deadac4442492e978

**Comment by dmitmel** [comment]
I'm closing this PR because supporting both Babel 6 and 7 will require moving `babel-loader` to `peerDependencies`.

**Comment by dmitmel** [comment]
I'm re-opening this PR because we should upgrade to `babel@7` in the v2.0 release.

**Comment by zcei** [comment]
@dmitmel can you point the PR towards the [release-2.0 branch](https://github.com/andywer/webpack-blocks/tree/release-2.0) please? 🙂

**Comment by dmitmel** [comment]
@zcei Done!

**Comment by andywer** [comment]
What‘s the status here? 😉

**Comment by dmitmel** [comment]
@andywer It seems like changes in this PR are outdated

**Comment by vlad-zhukov** [comment]
Outdated

**Pull request closed without merging**